<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbstractView</class>
 <widget class="QWidget" name="AbstractView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AbstractView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="separatorFuncButtons">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>26</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame#separatorFuncButtons {
     color: white;
     background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(128, 128, 128, 255), stop:0.5 rgba(0, 0, 0, 255), stop:0.95 rgba(64, 64, 64, 255), stop:1 rgba(200, 200, 200, 255));
 }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="viewCombo">
          <property name="styleSheet">
           <string notr="true">QComboBox {
  color: white;
  border-top: none;
  border-bottom: none;
  border-left: none;
  border-right-width: 1px;
  border-right-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 transparent, stop:0.5 rgba(200, 200, 200, 255), stop:1 transparent);
  border-right-style: solid;
  padding-left: 10px;
  padding-right: 10px;
}

QComboBox:editable {
  background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
  color: white;
  background: transparent;
  selection-background-color: lightgray;
  selection-color: black;
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
  color: white;
  selection-color: black;
}

QComboBox:on { /* shift the text when the popup opens */
  /*padding-top: 3px;
  padding-left: 4px;*/
}

QComboBox::drop-down {
  subcontrol-origin: padding;
  subcontrol-position: top left;
  width: 20px;

  border-right-width: 1px;
  border-right-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 transparent, stop:0.5 rgba(200, 200, 200, 255), stop:1 transparent);
  border-right-style: solid; 
}

QComboBox::down-arrow {
  image: url(:/Resources/Images/treeDownTriangleWhite.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
  top: 1px;
}</string>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="comboSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>11</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="funcCombo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
  color: white;
  border-top: none;
  border-bottom: none;
  border-left-width: 1px;
  border-left-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 transparent, stop:0.5 rgba(200, 200, 200, 255), stop:1 transparent);
  border-left-style: solid;
  border-right-width: 1px;
  border-right-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 transparent, stop:0.5 rgba(200, 200, 200, 255), stop:1 transparent);
  border-right-style: solid;
  padding-left: 10px;
  padding-right: 10px;
}

QComboBox:editable {
  background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
  color: black;
  background: transparent;
  selection-background-color: lightgray;
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
  color: black;
}

QComboBox:on { /* shift the text when the popup opens */
  /* padding-top: 3px;
  padding-left: 4px; */
}

QComboBox::drop-down {
  subcontrol-origin: padding;
  subcontrol-position: top left;
  width: 20px;

  border-right-width: 1px;
  border-right-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 transparent, stop:0.5 rgba(200, 200, 200, 255), stop:1 transparent);
  border-right-style: solid;
}

QComboBox::down-arrow {
  image: url(:/Resources/Images/treeDownTriangleWhite.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
  top: 1px;
}</string>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="buttonSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>62</width>
            <height>11</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="hSplitButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
     image: url(:/Resources/Images/horizontalsplit-16.png);
background-color: transparent;
}

QPushButton:pressed {
     image: url(:/Resources/Images/horizontalsplit-down-16.png);
}

QPushButton:flat {
     border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vSplitButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
     image: url(:/Resources/Images/verticalsplit-16.png);
background-color: transparent;
}

QPushButton:pressed {
     image: url(:/Resources/Images/verticalsplit-down-16.png);
}

QPushButton:flat {
     border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
     image: url(:/Resources/Images/closedock-16.png);
background-color: transparent;
}

QPushButton:disabled {
     image: url(:/Resources/Images/closedock-down-16.png);
}

QPushButton:pressed {
     image: url(:/Resources/Images/closedock-down-16.png);
}

QPushButton:flat {
     border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
